prijateli_tree/app/routers/games.py

```diff
@@ -484,6 +484,67 @@ def real_game_transition(
     return templates.TemplateResponse("real_game_transition.html", result)
 
 
+@router.get(
+    "/{game_id}/player/{player_id}/end_of_session",
+    response_class=HTMLResponse,
+)
+def end_of_session(
+    request: Request,
+    game_id: int,
+    player_id: int,
+    db: Session = Depends(get_db),
+) -> Response:
+    """
+    Function that returns the end of session page and
+    template.
+    """
+    game, player = get_game_and_player(game_id, player_id, db)
+    session_player = get_session_player_from_player(player, db)
+
+    # Get points and won games from session player
+    total_points = session_player.points
+    won_games = session_player.correct_answers
```

Comment:
we already call this correct_answers, so it will be easier to track if we keep the same name. Though, I prefer `n_correct_answers` personally as it makes it clear it's an integer.